lib/highlights-helper.coffee

```diff
@@ -70,6 +70,9 @@ scopesByFenceName =
   'ts': 'source.ts'
 
 module.exports =
-  scopeForFenceName: (fenceName) ->
+  scopeForFenceName: (fenceName, blockText) ->
     fenceName = fenceName.toLowerCase()
-    scopesByFenceName[fenceName] ? "source.#{fenceName}"
+    if fenceName is 'php' and not /<\?(php|=)/.test(blockText)
```

Comment:
I think it would be much faster to use `blockText.startsWith('<?php')`. regex should be avoided if possible.

Reply:
I agree, it's fixed